app/models/art.rb

```diff
@@ -1,10 +1,9 @@
 class Art < ApplicationRecord
   belongs_to :user
-  belongs_to :category
 
-  has_one_attached :photo
+  has_many_attached :photos
 
   # validations
   validates :title, :location, :description, :user, :category, :photo, presence: true
-  validates :price, :size, presence: true, numericality: { only_integer: true }, greater_than: 0
+  validates :price, presence: true, numericality: {only_float: true}, greater_than: 0
```

Comment:
:photo should be :photos

And we don't need numericality: {only_float: true} otherwise if someone wanna rent something for a round number he can't unless he do 10.0€. That's just not necessary. It can be an integer too.
The greather_than that you have already ensures that will be a number higer than 0